package.json

```diff
@@ -70,9 +70,6 @@
     "typescript": "^4.1.5"
   },
   "peerDependencies": {
-    "react": "^17.0.1",
-    "react-dom": "^16.14.0",
-    "react-router-dom": "^5.2.0",
```

Comment:
These three still need to be in `peerDependencies`. We just need to change `react-dom` to use the same version as React.